src/color/converters/jsx/Text.tsx

```diff
@@ -42,7 +42,7 @@ export const TextJSXConverter: JSXConverters<SerializedTextNode> = {
 			let match = node.style.match(/background-color: ([^;]+)/)
 			match && (style.backgroundColor = match[1]);
 
-			match = node.style.match(/color: ([^;]+)/)
+			match = node.style.match(/^color: ([^;]+)/)
```

Comment:
What if the color is not the first style? maybe we should check if there's no letter and no dash before `color`

Reply:
```suggestion
			match = node.style.match(/;?color: ([^;]+)/)
```
What if we do that instead and add a trim to remove the ";"

Reply:
Ok, we could go that way, but then we should also check if there’s a whitespace after the semicolon

Reply:
Hello, i have pushed a new commit :) 
Let me know what you think